File: withFunctions/usingStatic.java
package springIntro;

public class usingStatic {
	static int a= 9;
	static int go() {
		a += 5;
		return a;
	}
	
	public static void main(String[] args) {
		int b = go();
		System.out.println("First Result = " + b);
		b = go();
		System.out.println("Second Result = " + b);

	}

}

File: withFunctions/usingStatic.cpp
#include<iostream>
using namespace std;
int go(){
	static int a = 9;
	a += 5;
	return a;
}
int main(){
	int b = go();
	cout << "Result = " << b << endl;
	b = go();
	cout << "Result = " << b;
	return 0;
}
